package.json

```diff
@@ -316,6 +329,14 @@
         },
         "menus": {
             "commandPalette": [
+                {
```

Comment:
I changed the when clauses for both the commands to: `view == azureAppService && viewItem == trialApp`. I realize this might be too specific (user has to be focused on the trial app item?) for these commands to show up in the palette now. 

Reading from when-clause docs [here](https://code.visualstudio.com/docs/getstarted/keybindings#_when-clause-contexts)

I think this is the *simple* only way to:

> disallow depending on when there is/is not a trial app attached

without adding an extension setting and referencing it with "config." in the when clause.

Reply:
As for the import command, I think that this should never be in the palette because the only way to import a trial app (for now) is through the browser and an external Uri.

Reply:
Sadly, there is no way to import based on the hostname URL, because we need their login session from when they authenticate in the browser.